ProjectProtocol/project-protocol-web
# [PRO 293] resource details route and page

Changes
---
* Adds a `get(id: string)` function to api/resources.ts for fetching an individual resource.
* Adds a `ResourceView` page component and route to router.
* Renders a `ResourceCard` to display resource info in `ResourceView`.

Testing
---
* Check out the [live preview](https://deploy-preview-117--fabulous-sprite-a61e48.netlify.app/)
* Navigate to `resources/:id` (replace id with number). Verify the screen looks good and up voting/down voting works.
* The rest of the site/navigation is working properly.
